ita-social-projects/GreenCityUBS
# Card #4495 - Order History fixed for canceled orders

# Card #4495 - Order History fixed for canceled orders `READY_FOR_REVIEW`
Order History fixed for canceled orders

## Summary Of Changes :fire:
The Order History in Order summary wasn't complete because when the order was payed with bonuses, it didn't updated when was cancelled, and now it saves in history when an order was cancelled.

## Issue Link :clipboard:
[Card #4495](https://github.com/ita-social-projects/GreenCity/issues/4495)

## Added
* Now saves the history when you cancel an order.

## Changed
* The test now verify that the saveEvent is called 

## How to test :clipboard:
1. First you need to run GreenCityClient
2. Create a completely order and paid all with bonuses
3. Later you need to cancel this order
4. Click the user name and then click on UBS-admin
5. Look for the order that you cancelled
6. Click on Order History
7. Then you would be able to see the History there

# PR Checklist Forms
